File: app/src/main/java/social/entourage/android/map/filter/MapFilterInterface.java
package social.entourage.android.map.filter;

/**
 * Created by mihaiionescu on 17/05/16.
 */
public interface MapFilterInterface {

    String getTypes();
    int getTimeFrame();
    boolean showPastEvents();

    void entourageCreated();
    void validateCategories();

}

File: app/src/main/java/social/entourage/android/map/filter/MapFilterInterface.kt
package social.entourage.android.map.filter

/**
 * Created by mihaiionescu on 17/05/16.
 */
interface MapFilterInterface {
    fun getTypes(): String?
    fun getTimeFrame(): Int
    fun showPastEvents(): Boolean
    fun entourageCreated()
    fun validateCategories()
    fun isDefaultFilter(): Boolean
    fun setDefaultValues(isProUser: Boolean = false)
}